client/my-sites/stats/activity-log/activity-log-empty.js

```diff
@@ -0,0 +1,16 @@
+/** @format */
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { localize } from 'i18n-calypso';
+
+function ActivityLogEmpty( { translate } ) {
+	return (
+		<div>
+			{ translate( 'No activity for this period' ) }
```

Comment:
Hi! I've found a possible matching string that has already been translated **24** times:
`translate( 'No activity this period' )` *ES Score: 12*
See 1 additional suggestion in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/add/activitylog-placeholder)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).